--- kegs.0.91/src/joystick_driver.c.ark	2012-01-04 18:47:08.389576364 +0100
+++ kegs.0.91/src/joystick_driver.c	2012-01-04 18:47:21.501412444 +0100
@@ -29,7 +29,7 @@ extern int g_paddle_buttons;
 extern int g_paddle_val[];
 
 
-const char *g_joystick_dev = "/dev/js0";	/* default joystick dev file */
+const char *g_joystick_dev = "/dev/input/js0";	/* default joystick dev file */
 #define MAX_JOY_NAME	128
 
 int	g_joystick_native_fd = -1;
